### PR TITLE
[DO NOT MERGE] Version 2.0.3

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,85 @@ The MSAL library for Android gives your app the ability to use the [Microsoft Cl
 
 > Looking for developers interested in providing early feedback on a x-platform implementation of MSAL written in C++ and Java, callable from Java, Kotlin and C++.  If you're interested please please contact shoatman@microsoft.com.
 
+***2020-11-09*** New updates with [MSAL 2.0.3](https://github.com/AzureAD/microsoft-authentication-library-for-android/releases/tag/v2.0.3)
```
Comment (shahzaibj) [inline README.md:19]: I don't think all these updates are necessary.....just an overview of what is new with each major release should be sufficient.....this info is anyways available in the releases page as well as the changelog

Comment (AdamBJohnsonx) [reply README.md]: cut to 2.0.0

File: msal/versioning/version.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #Wed Aug 01 15:24:11 PDT 2018
-versionName=2.0.2
+versionName=2.0.3
```
Comment (shahzaibj) [inline msal/versioning/version.properties:2]: we want this to be RC1

Comment (AdamBJohnsonx) [inline msal/versioning/version.properties:2]:
```suggestion
versionName=2.0.3-RC1
```

Comment (AdamBJohnsonx) [reply msal/versioning/version.properties]: Added.

File: README.md [hunk 3]
```diff
@@ -16,6 +16,16 @@ The MSAL library for Android gives your app the ability to use the [Microsoft Cl
 
 > Looking for developers interested in providing early feedback on a x-platform implementation of MSAL written in C++ and Java, callable from Java, Kotlin and C++.  If you're interested please please contact shoatman@microsoft.com.
 
+***11/09/2020***
+ - Android changes for SDK30, see [the android developers notice].(https://android-developers.googleblog.com/2020/07/preparing-your-build-for-package-visibility-in-android-11.html)
```
Comment (AdamBJohnsonx) [inline README.md:20]:
```suggestion
 - Android changes for SDK30, see [the android developers notice](https://android-developers.googleblog.com/2020/07/preparing-your-build-for-package-visibility-in-android-11.html).
```

**Pull request merged**